### PR TITLE
nvimpager: update to 0.9

### PR DESCRIPTION
**Pull request by adrianisuru**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/nvimpager/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'nvimpager'
 pkgname=nvimpager
-version=0.7
-revision=2
+version=0.9
+revision=3
```
Comment (ericonr) [inline srcpkgs/nvimpager/template:4]: You need to set `revision=1` when updating a package.

File: srcpkgs/nvimpager/template [hunk 2]
```diff
@@ -10,15 +10,14 @@ build_style=gnu-makefile
...
+depends="neovim>=0.4.0 bash"
 short_desc="Wrapper to use neovim as a pager to view manpages, diffs, etc"
 maintainer="Adrian Herath <adrianisuru@gmail.com>"
 license="BSD-2-Clause"
 homepage="https://github.com/lucc/nvimpager"
 distfiles="${homepage}/archive/v${version}.tar.gz
  https://www.vim.org/scripts/download_script.php?src_id=14498>AnsiEsc.vba.gz"
-checksum="cb4eee0ef2569412b0528eacd7efd6af960bebc11d8749258d8ba83c975ebf4b
- a80d8c95a2e497e190831f1b580205653c595c6f51b48d4cb89fb33eaea5ed03"
+checksum="ae2d99d59bd43d96e12a07ffa32c3f120636163308c2b6c1da43052bc8bce1a0 a80d8c95a2e497e190831f1b580205653c595c6f51b48d4cb89fb33eaea5ed03"
```
Comment (ericonr) [inline srcpkgs/nvimpager/template:20]:
```suggestion
checksum="ae2d99d59bd43d96e12a07ffa32c3f120636163308c2b6c1da43052bc8bce1a0
 a80d8c95a2e497e190831f1b580205653c595c6f51b48d4cb89fb33eaea5ed03"
```

File: srcpkgs/nvimpager/template [hunk 3]
```diff
@@ -10,15 +10,14 @@ build_style=gnu-makefile
...
 make_build_args="PREFIX=/usr"
 hostmakedepends="neovim $(vopt_if man pandoc)"
-depends="neovim>=0.3.0 bash"
+depends="neovim>=0.4.0 bash"
 short_desc="Wrapper to use neovim as a pager to view manpages, diffs, etc"
 maintainer="Adrian Herath <adrianisuru@gmail.com>"
 license="BSD-2-Clause"
 homepage="https://github.com/lucc/nvimpager"
 distfiles="${homepage}/archive/v${version}.tar.gz
  https://www.vim.org/scripts/download_script.php?src_id=14498>AnsiEsc.vba.gz"
```
Comment (ericonr) [inline srcpkgs/nvimpager/template:19]: Should we update this at some point?

Comment (adrianisuru) [reply srcpkgs/nvimpager/template:19]: Which "this" are you talking about? The depends? Sorry I'm not clear on what you are asking.

Comment (ericonr) [reply srcpkgs/nvimpager/template:19]: Sorry, I meant `src_id=14498` in the link.

File: srcpkgs/nvimpager/template (hunk 2 above)
Comment (ericonr) [reply srcpkgs/nvimpager/template:20]: This still needs to be fixed.

File: srcpkgs/nvimpager/template (hunk 3 above)
Comment (ericonr) [reply srcpkgs/nvimpager/template:19]: So, given https://github.com/lucc/nvimpager/blob/develop/makefile , I don't see this file being used anymore? Maybe we should remove from package.

Comment (ericonr) [reply srcpkgs/nvimpager/template:19]: We should definitely remove, they no longer use it.

**Pull request closed without merging**